.github/workflows/main.yml

```
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [stable, unstable, development-target]
+    container:
+      image: ghcr.io/elementary/docker:${{ matrix.version }}
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Dependencies
+        run: |
+          apt update
+          apt install -y libgranite-dev meson valac
+      - name: Build
+        env:
+          DESTDIR: out
+        run: |
+          meson build
```

Because implicit `setup` is now deprecated.

```suggestion
          meson setup build
```